gcp_variant_transforms/testing/integration/run_tests_common.py

```diff
@@ -81,9 +86,50 @@ def _handle_failure(self, response):
         raise TestCaseFailure(
             'No traceback. See logs for more information on error.')
 
-  @abstractmethod
-  def validate_result(self):
-    raise NotImplementedError
+  def print_errors(self):
```

Comment:
Is this function ever called if there is a failure in one of the tests? I mean, wouldn't the whole program fail before calling this function because of the exception raised in `_handle_failure`?